package.json

```diff
@@ -85,7 +85,7 @@
         "build": "cd lib && rm -rf build && npx tsc -p tsconfig.json",
         "watch": "cd lib && npx tsc -w -p tsconfig_dev.json",
         "pretty": "npx pretty-quick .",
-        "build-pretty": "npm run build && npm run pretty",
+        "build-pretty": "npm run build && npm run pretty && npm run pretty",
```

Comment:
why are we running pretty twice?

Reply:
There was this annoying issue where after running pretty once pretty-check would still fail for some files (the same issue happened in supertokens-node as well). There was no clear reason as to why it was happening and running pretty again solved it so went with this approach for now.

Reply:
Hmm, what changes between the two runs? We should investigate this a bit. Running it twice is fine for the time being, but we should create an issue/TODO about this.

Reply:
So one case is that the first time we run pretty, the thirdpartyemailpassword/index.js changes in a way that pretty-check will fail. To fix this you would have to split the code in multiple lines but when you do that pretty changes it back to one line which makes it too long for pretty-check.

This looks like a case where prettier rules conflict with each other and one takes precedence. This will eventually be a problem in all TS SDKs where we use prettier.